ui-v2/lib/startup/templates/body.html.js

```diff
@@ -41,5 +41,8 @@ module.exports = ({ appName, environment, rootURL, config }) => `
       }
     };
   </script>
+  {{ range .ExtraScripts }}
+    <script src="{{.}}"></script>
+  {{ end }}
```

Comment:
I realised this needs to be conditional on `environment === production` otherwise it prints this junk into the page in UI dev mode.